package.json

```diff
@@ -173,6 +173,7 @@
     "react-table": "^7.0.0",
     "react-table-sticky": "^1.1.0",
     "react-timeago": "^4.4.0",
+    "react-toastify": "^6.0.9",
```

Comment:
I agree about the aesthetics of `react-toastify-notifications`. I decided on the current package because of the hack that I would have to write to get it to work but since it's not an issue, I will swap it out.